src/kOS.Safe/Screen/ScreenBuffer.cs

```diff
@@ -203,12 +202,17 @@ private void PrintLine(string textToPrint)
             MoveColumn(textToPrint.Length);
         }
 
+		public void HudTxt(string textToHud, int delay, int style, int size, string color, int mirror)
+		{
+		}
+
```

Comment:
As far as I can tell, I don't think this method is actually being called by anywhere.

Was it something you stared thinking you'd use, and then you changed your mind and it's just leftover in the code from an older idea (meaning we should delete it), or is it a stub you plan to use in a future update (meaning we shouldn't delete it) ?